src/storage/mod.rs

```diff
@@ -57,66 +57,65 @@ impl Mutation {
 
 use kvproto::kvrpcpb::Context;
 
+pub enum StorageCb {
+    Result(Callback<()>),
```

Comment:
Result and following MultiResults confuse me, what's result for? does Value and KvPairs are not result?